meinberlin/assets/scss/components_user_facing/_dropdown.scss

```diff
@@ -0,0 +1,46 @@
+.dropdown {
+    position: relative;
+}
+
+.dropdown__toggle {
+    i {
+        font-size: 1.25em;
+    }
+}
+
+.dropdown__menu {
+    position: absolute;
+    display: none;
+    right: 0;
+    white-space: nowrap;
+    border: 1px solid $text-color;
+    background-color: $white;
+    flex-direction: column;
+    justify-content: space-between;
+    box-shadow: 0 0 5px rgba(100, 100, 100, 0.5);
+
+    ul {
+        margin-bottom: 0;
+    }
+
+    @media print {
+        display: none;
+    }
+}
+
+.dropdown__item {
+    padding: $spacer / 2 $spacer;
```

Comment:
can we use `0.5 * $spacer` here? otherwise we'll get the deprecation warning from sass:
```
Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($spacer, 2) or calc($spacer / 2)
```